recipes/sipros/build.sh

```
@@ -0,0 +1,13 @@
+set -e
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)